packages/backend-tasks/package.json

```diff
@@ -51,8 +55,5 @@
     "@backstage/cli": "workspace:^",
     "wait-for-expect": "^3.0.2"
   },
-  "files": [
-    "dist",
-    "migrations/**/*.{js,d.ts}"
-  ]
+  "deprecated": true
```

Comment:
Hmm, is this a thing? Wouldn't have expected a boolean. Where did you find it?

Reply:
oh is that what's successfully resulting in the bar at the top of https://www.npmjs.com/package/@backstage/plugin-scaffolder-backend-module-bitbucket ?

Reply:
Just seems it's not known [by node](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions), nor [by npm](https://docs.npmjs.com/cli/v8/configuring-npm/package-json) 🤔

Reply:
Change reverted as we are going to configure that in the npm registry:
https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions